client/my-sites/people/viewers-list/invite-button.jsx

```diff
@@ -0,0 +1,13 @@
+import { Button, Gridicon } from '@automattic/components';
+import { localize } from 'i18n-calypso';
+
+const InviteButton = ( { siteSlug, translate } ) => {
+	return (
+		<Button primary={ true } href={ `/people/new/${ siteSlug }` }>
```

Comment:
`={ true }` is redundant:

```suggestion
		<Button primary href={ `/people/new/${ siteSlug }` }>
```